google_guest_agent/windows_accounts_test.go

```diff
@@ -218,25 +218,60 @@ func TestCompareAccounts(t *testing.T) {
 
 func TestRemoveExpiredKeys(t *testing.T) {
 	var tests = []struct {
-		key   string
-		valid bool
+		key      string
+		valid    bool
+		userKeys string
```

Comment:
see comment below for suggested testing:

```suggestion
		expectedValid    int
```